atd-vze/src/Components/GridExportData.js

```diff
@@ -68,15 +68,18 @@ const GridExportData = ({ query, columnsToExport, totalRecords }) => {
     // Move nested keys to top level object (CSVLink uses each top level key as a column header)
     const flattenRow = (row, flattenedRow) => {
       Object.entries(row).forEach(([columnName, columnValue]) => {
-        // Ignore __typename (contains table name which is already in filename)
         if (columnName === "__typename") {
+          // Ignore __typename (contains table name which is already in filename)
           return;
         } else if (Array.isArray(columnValue)) {
           // If value is array, recursive call and handle objects in array
           flattenRow(columnValue, flattenedRow);
         } else if (typeof columnValue === "object" && columnValue !== null) {
           // If value is object, recursive call and handle k/v pairs in object
           flattenRow(columnValue, flattenedRow);
+        } else if (columnName === "death_cnt") {
+          // Rename death_cnt column to cris_death_cnt at the request of VZ Team
```

Comment:
@JaceDeloney, is this rename something that should be happening only for the CSV export? Should we be doing this at a system/database level?

Reply:
nvm, @tylerwhitson let me know its only for CSVs